content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -111,4 +111,9 @@ To install Minikube manually on windows using [Windows Installer](https://docs.m
 
 {{% /capture %}}
 
+## Cleanup everything to start fresh
+If you have installed minikube before and when running `minikube start` is throwing error `machine does not exist` then simply wipe existing configuration files.
```

Comment:
Not just `machine does not exist` error, I faced another issue where removing the config file helped. 

- `Minikube start` used to get stuck at starting the VM, and when checked the logs, I would only see a timeout error in a never-ending loop for a particular API call (Not sure which one particularly) 

Maybe can we frame this a little generic?

Reply:
@iamneha This needs better wording. Consider:
> ## Starting fresh after a previous installation
> 
> If you have previously installed minikube, and run:
> ```sh
> minikube start
> ```
> If this command returns an error:
> ```sh
> machine does not exist
> ```
> You need to wipe the configuration files: